tools/tasks/mediaTasks.js

```diff
@@ -9,7 +9,8 @@ module.exports = function(gulp, config) {
 
     const jsonMinify = () => gulp.src([
             path.join(OPTIONS.DIR.DIST_AIRPORTS, '**/*.json'),
-            path.join(OPTIONS.DIR.DIST_AIRPORTS, '**/*.geojson')
+            path.join(OPTIONS.DIR.DIST_AIRPORTS, '**/*.geojson'),
+            '!' + path.join(OPTIONS.DIR.DIST_AIRPORTS, 'airportLoadList*.json')
```

Comment:
why don't we want this minified?

Reply:
The original `airportLoadList.js` was omitted from minifying + bundling in order to keep it accessible for modification without the need for rebuilding:

https://github.com/openscope/openscope/blob/67f23ecfa27e73eb698ee7fe7c008ac03fdb8d3e/assets/airports/airportLoadList.js#L2-L7

Likewise do not minify this json + its schema, preserving human readability and ability to easily edit the live version at `public/assets/airports/airportLoadList.json`. I think the ideas is to enable airport contributors to test their changes without having to rebuild the app -- all that's needed is to plonk a copy of new stuff into `public/assets/airports/` and edit the airport load list accordingly.

Reply:
excellent.  that is still the idea, our concern for contributors.  so as long as thats still possible we're good to go